### DISCUSSION TITLE
How to show/display video/media controls

### DISCUSSION THREAD

**Discussion by lextra2** [uBlockOrigin/uBlock-issues#3422] [Q&A]
On instagram.com for example, there are no video controls.
I'd like to force showing the default html5 media controls.
I know that the `<video>` element requires `controls="true"`

Is this possible to do with uBlock Origin?

**Comment by gorhill** [comment]
Try:

    host.name##+js(set-attr, controls ,true)

**Reply by lextra2** [reply]
Doesn't work. (Unless Instagram overrides it _again_ and I need to find and block that script?)
[I did some more research](https://stackoverflow.com/a/14161560), and apparently "true" or "false" arent allowed for the "controls" attribute, even though its a boolean...

So I need a way to add that attribute to the video element.

Edit: Logically I also tried your suggestion without the "true" part, which unfortunately also didn't work.

**Reply by gorhill** [reply]
Oops forgot the selector, was working from memory.

**Comment by lextra2** [comment]
Fixed it.

`instagram.com##+js('set-attr', 'video', 'controls', 'true')`

Now I just need to hide their custom media elements and I'm done.
